--- asdcplib-1.7.40/configure	2011-02-05 01:17:49.000000000 -0500
+++ configure	2011-02-05 01:26:03.000000000 -0500
@@ -20689,7 +20689,7 @@
        }; then
 
                 OPENSSL_LDFLAGS="$openssl_ldflags"
-                OPENSSL_LIBS="$openssl_libs"
+                OPENSSL_LIBS="$openssl_prefix/lib/libssl.a $openssl_prefix/lib/libcrypto.a"
                 openssl_lib_found="yes"
                 { $as_echo "$as_me:$LINENO: result: found" >&5
 $as_echo "found" >&6; }

--- asdcplib-1.7.40/src/KM_fileio.cpp       2011-02-08 23:06:39.000000000 -0500
+++ KM_fileio.cpp       2011-02-09 02:04:04.000000000 -0500
@@ -54,7 +54,7 @@
   int   iov_len;
 };
 #else
-# if defined(__linux__)
+# if defined(__linux__) || defined(__CYGWIN__)
 #   include <sys/statfs.h>
 # else
 #  include <sys/param.h>
~                             
